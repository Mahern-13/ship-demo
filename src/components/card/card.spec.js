import { shallow } from "enzyme";
import React from "react";
import { Default, Primary, Warning, Danger } from "./index";

describe("Default", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Default header="header" children="children" />);

    expect(wrapper.exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("Primary", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Primary header="header" children="children" />);

    expect(wrapper.exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("Warning", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Warning header="header" children="children" />);

    expect(wrapper.exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("Danger", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Danger header="header" children="children" />);

    expect(wrapper.exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
