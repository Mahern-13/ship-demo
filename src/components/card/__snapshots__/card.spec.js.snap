// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Danger renders the component 1`] = `ShallowWrapper {}`;

exports[`Default renders the component 1`] = `ShallowWrapper {}`;

exports[`Primary renders the component 1`] = `ShallowWrapper {}`;

exports[`Warning renders the component 1`] = `ShallowWrapper {}`;
